saleor/graphql/payment/mutations.py

```diff
@@ -686,13 +688,13 @@ class TransactionEventInput(graphene.InputObjectType):
     )
     name = graphene.String(description="Name of the transaction.")
     type = graphene.Field(
-        TransactionActionEnum,
+        TransactionEventActionTypeEnum,
         description=(
             "The transaction action that is related to this event." + ADDED_IN_38
         ),
     )
-    amount = graphene.Field(
-        PositiveDecimal, description="The amount related to this event." + ADDED_IN_38
+    amount = scalars.Decimal(
```

Comment:
I guess you could even use `PositiveDecimal` here, would that make sense?

Reply:
It was used already before, but we don't want to limit someone in specific payment use case.
> Some fields has been changed to Decimal, as we want to be more flexible in new payment flow and want to cover more specific scenarions.

Reply:
👍